changelog/README.md

```diff
@@ -18,6 +18,7 @@
 - [improvement] JAVA-1950: Log server side warnings returned from a query.
 - [improvement] JAVA-2123: Allow to use QueryBuilder for building queries against Materialized Views.
 - [bug] JAVA-2082: Avoid race condition during cluster close and schema refresh.
+- [bug] JAVA-2174: quote empty identifier
```

Comment:
Also the correct changelog entry is `Metadata.needsQuote should accept empty strings.` (with a final period).